tests/unittests/audio/test_sa_sdr.py

```diff
@@ -0,0 +1,147 @@
+# Copyright The Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from collections import namedtuple
+from functools import partial
+
+import pytest
+import torch
+from torch import Tensor
+from torchmetrics.audio import SourceAggregatedSignalDistortionRatio
+from torchmetrics.functional.audio import (
+    scale_invariant_signal_distortion_ratio,
+    signal_noise_ratio,
+    source_aggregated_signal_distortion_ratio,
+)
+
+from unittests import BATCH_SIZE, NUM_BATCHES
+from unittests.helpers import seed_all
+from unittests.helpers.testers import MetricTester
+
+seed_all(42)
+
+Time = 100
```

Comment:
can we rather have it as constant and include it in the title units, like seconds?

Reply:
the unit of the time dimension for audio metrics is either number of samples or the number of frames. I will add its unit in the comment.